manuscript/manuscript.Rmd

```diff
@@ -23,18 +23,17 @@ authors:
     email: cboettig@berkeley.edu (corresponding author)
 abstract: |
   Can machine learning help us make better decisions about a changing planet?
-  In this paper, we illustrate and discuss the potential of a promising 
-  corner of machine learning known as _reinforcement learning_ (RL) to help tackle the
-  most challenging conservation decision problems. RL is uniquely
-  well suited to conservation and global change challenges for three reasons: (1)
-  RL explicitly focuses on designing an agent who _interacts_ with an environment which is dynamic and uncertain,
-  (2) RL approaches do not require massive amounts of data, (3) RL approaches would 
-  utilize rather than replace existing models, simulations, and the knowledge they contain.
-  We provide a conceptual and technical introduction to RL and its relevance to ecological
-  and conservation challenges, including examples of a problem in setting
-  fisheries quotas and in managing ecological tipping points.
-  Four appendices with annotated code provide a tangible introduction to researchers 
-  looking to adopt, evaluate, or extend these approaches.
+  In this paper, we illustrate and discuss the potential of a promising corner of machine learning known as _reinforcement learning_ (RL) to help tackle the most challenging conservation decision problems.
+  RL is uniquely well suited to conservation and global change challenges for three reasons: 
+  (1) RL explicitly focuses on designing an agent who _interacts_ with an environment which is dynamic and uncertain,
+  (2) RL approaches do not require massive amounts of data, 
+  (3) RL approaches would utilize rather than replace existing models, simulations, and the knowledge they contain.
+  While this potential may be compelling, the challenges involved in successfully deploying RL-based management to realistic scenarios are formidable --
+  the required expertise and computational cost may place these applications may be beyond the reach of all but international technology firms.
```

Comment:
"the required expertise and computational cost may place these applications beyond the reach of all but international technology firms."